messages/run.js

```diff
@@ -1,101 +1,100 @@
 module.exports = {
-	"commandDescription": "evaluate a selection of pathless rules against a codebase",
-	"commandDescriptionLong": `Scan codebase with all pathless rules by default
-	or with a chosen set of rules if using rulename/category/ruleset filters.
-	You can choose the format of output and decide between printing the results directly
-	or as contents of a file that you provide with --outfile flag.`,
+	"commandDescription": "scan a codebase with a selection of rules",
+	"commandDescriptionLong": `Scans a codebase with a selection of rules. You can scan the codebase with all the rules in the registry, or use parameters to filter the rules based on rulename, category, or ruleset. You can specify the format of the output, such as XML or JUnit. You can print the output to the console (default) or to a file using the --outfile parameter.`,
 	"flags": {
-		"categoryDescription": "categor(ies) of rules to run",
-		"categoryDescriptionLong": "One or more categories of rules to run. Multiple values can be specified as a comma-separated list.",
-		"rulesetDescription": "[deprecated] ruleset(s) of rules to run",
-		"rulesetDescriptionLong": "[Deprecated] One or more rulesets to run. Multiple values can be specified as a comma-separated list.",
-		"targetDescription": "location of source code",
-		"targetDescriptionLong": "Source code location. May use glob patterns. Multiple values can be specified as a comma-separated list",
-		"formatDescription": "format of results",
+		"categoryDescription": "one or more categories of rules to run",
+		"categoryDescriptionLong": "One or more categories of rules to run. Specify multiple values as a comma-separated list.",
+		"rulesetDescription": "[deprecated] ruleset to run",
+		"rulesetDescriptionLong": "[deprecated] One or more rulesets to run. Specify multiple values as a comma-separated list.",
+		"targetDescription": "source code location",
+		"targetDescriptionLong": "Source code location. May use glob patterns. Specify multiple values as a comma-separated list.",
+		"formatDescription": "specify results output format",
 		"formatDescriptionLong": "Specifies output format with results written directly to the console.",
-		"outfileDescription": "location of output file",
-		"outfileDescriptionLong": "Write output to a file.",
-		"envDescription": "JSON-formatted string, overrides ESLint's default environment variables",
-		"envDescriptionLong": "JSON-formatted string. Overrides ESLint's default environmental variables.",
+		"outfileDescription": "write output to a file",
+		"outfileDescriptionLong": "Writes output to a file.",
+		"envDescription": "[deprecated] override ESLint's default environment variables, in JSON-formatted string",
```

Comment:
@jfeingold35 Are envDescription, envDescriptionLong deprecated? They aren't noted to be in the original descriptions nor in the Google doc?

Reply:
@teresa-allen-sfdc , they are. We  have the deprecation message at line 17.

Reply:
@jfeingold35 No, what I'm saying is in our approved UI text doc, on page 19, envDescription and envDescriptionLong are NOT noted as being deprecated so we have a mismatch between the approved UI text doc and this code. But you're saying both are deprecated so I need to update the approved UI text doc?
https://docs.google.com/document/d/1-PJ-Mj3GDGcXimlAJeAZjWuk_XIfsEJz8cEM1pS7Zdk/edit#